### PR TITLE
Various Fixes

### PR DESCRIPTION
**Pull request by Pascalmh**
Fixed stuff I found with the Inspector Tool from phpStorm - mostly typos, stuff that changed (image width / height e.g.) and stuff that probably broke while refactoring
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/Phhere%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/143%23issuecomment-69495340%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222015-01-11T13%3A59%3A44Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%200f458f8f9c3cdf94216d19fd997ef7d40e37917b%20admin/settings.php%2014%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/143%23discussion_r22766586%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Ther%20is%20a%20%3C%20/select%3E%20to%20much%22%2C%20%22created_at%22%3A%20%222015-01-11T12%3A57%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-01-11T13%3A13%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20admin/settings.php%3AL426-433%22%7D%2C%20%22Pull%200f458f8f9c3cdf94216d19fd997ef7d40e37917b%20templates/comments_add_visitor.html%206%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/143%23discussion_r22765793%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20think%20that%20this%20is%20wrong%2C%20because%20the%20name%20is%20used%20as%20anchor%20in%20urls%20%23addcomment%22%2C%20%22created_at%22%3A%20%222015-01-11T10%3A31%3A41Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22We%20may%20achieve%20the%20same%20effect%20by%20making%20the%20header%20elements%20themselves%20the%20anchors%3A%5Cr%5Cn%5Cr%5Cn%60%60%60xml%5Cr%5Cn%3CH1%3ETable%20of%20Contents%3C/H1%3E%5Cr%5Cn%3CP%3E%3CA%20href%3D%5C%22%23section1%5C%22%3EIntroduction%3C/A%3E%3CBR%3E%5Cr%5Cn...the%20rest%20of%20the%20table%20of%20contents...%5Cr%5Cn...the%20document%20body...%5Cr%5Cn%3CH2%20id%3D%5C%22section1%5C%22%3EIntroduction%3C/H2%3E%5Cr%5Cn...section%201...%5Cr%5Cn%60%60%60%5Cr%5Cnhttp%3A//www.w3.org/TR/html401/struct/links.html%23h-12.1.1%20-%20the%20hr-Tag%20does%20not%20have%20the%20name-Attribute.%22%2C%20%22created_at%22%3A%20%222015-01-11T12%3A27%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%2C%20%7B%22body%22%3A%20%22What%20about%20adding%20the%20name%20to%20the%20form%20%3F%22%2C%20%22created_at%22%3A%20%222015-01-11T13%3A00%3A47Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-01-11T13%3A13%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20templates/comments_add_visitor.html%3AL35-42%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/143%23issuecomment-69495340%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/143%23discussion_r22765793%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/143%23discussion_r22766367%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/143%23discussion_r22766586%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/143%23discussion_r22766608%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/143%23discussion_r22766699%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/143%23discussion_r22766700%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/Phhere'></a>

- [x] <a href='#crh-comment-Pull 0f458f8f9c3cdf94216d19fd997ef7d40e37917b templates/comments_add_visitor.html 6'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/143#discussion_r22765793'>File: templates/comments_add_visitor.html:L35-42</a></b>
- <a href='https://github.com/Phhere'></a> I think that this is wrong, because the name is used as anchor in urls #addcomment
- <a href='https://github.com/Pascalmh'></a> We may achieve the same effect by making the header elements themselves the anchors:
```xml
<H1>Table of Contents</H1>
<P><A href="#section1">Introduction</A><BR>
...the rest of the table of contents...
...the document body...
<H2 id="section1">Introduction</H2>
...section 1...
```
http://www.w3.org/TR/html401/struct/links.html#h-12.1.1 - the hr-Tag does not have the name-Attribute.
- <a href='https://github.com/Phhere'></a> What about adding the name to the form ?
- [x] <a href='#crh-comment-Pull 0f458f8f9c3cdf94216d19fd997ef7d40e37917b admin/settings.php 14'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/143#discussion_r22766586'>File: admin/settings.php:L426-433</a></b>
- <a href='https://github.com/Phhere'></a> Ther is a < /select> to much

<a href='https://www.codereviewhub.com/webSPELL/webSPELL-4.2.3/pull/143?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL-4.2.3/pull/143?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL-4.2.3/pull/143?approve=0'></a>&nbsp;<a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/143'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by derchrisuk** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/143'>Refresh</a>.<a href='#crh-update-none'></a>

File: templates/comments_add_visitor.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
     -->
 </script>
 
-<hr name="addcomment">
+<hr id="addcomment">
 <form id="post" name="post" action="comments.php" method="post" onsubmit="return chkFormular()" class="form-horizontal">
```
Comment (Phhere) [inline templates/comments_add_visitor.html]: I think that this is wrong, because the name is used as anchor in urls #addcomment

Comment (Pascalmh) [inline templates/comments_add_visitor.html]: We may achieve the same effect by making the header elements themselves the anchors:

```xml
<H1>Table of Contents</H1>
<P><A href="#section1">Introduction</A><BR>
...the rest of the table of contents...
...the document body...
<H2 id="section1">Introduction</H2>
...section 1...
```
http://www.w3.org/TR/html401/struct/links.html#h-12.1.1 - the <hr>-Tag does not have the name-Attribute.

File: admin/settings.php [hunk 2]
```diff
@@ -426,7 +426,7 @@
             </tr>
             <tr>
                 <td align="right"><select name="captcha_math" onmouseover="showWMTT('id47')"
-                                          onmouseout="hideWMTT()"/><?php echo $captcha_style; ?></select></td>
+                                          onmouseout="hideWMTT()"></select><?php echo $captcha_style; ?></select></td>
```
Comment (Phhere) [inline admin/settings.php]: Ther is a </select> to much

File: templates/comments_add_visitor.html (hunk 1 above)
Comment (Phhere) [inline templates/comments_add_visitor.html]: What about adding the name to the form ?

File: admin/settings.php (hunk 2 above)
Comment (Pascalmh) [inline admin/settings.php]: :+1:<a href='#crh-update-none'></a>

File: templates/comments_add_visitor.html (hunk 1 above)
Comment (Pascalmh) [inline templates/comments_add_visitor.html]: :+1:<a href='#crh-update-complete'></a>

**Comment by Phhere** [comment]
:shipit:

**Pull request merged**